build.gradle

```diff
@@ -1226,15 +1226,27 @@ project(':generator') {
 }
 
 project(':clients') {
+  apply plugin: 'com.github.johnrengelman.shadow'
+
   archivesBaseName = "kafka-clients"
 
+  shadowJar {
+    zip64 = true
```

Comment:
is this actually required? we haven't had to use it elsewhere. Can you add a comment to explain which packages require this?

Reply:
we probably don't... originally I thought we would need this for larger project.